airflow/models/taskmixin.py

```diff
@@ -115,24 +117,39 @@ def __rlshift__(self, other: DependencyMixin | Sequence[DependencyMixin]):
         self.__rshift__(other)
         return self
 
+    @abstractmethod
+    def add_to_taskgroup(self, task_group: TaskGroup) -> None:
+        """Add the task to the given task group."""
+        raise NotImplementedError()
+
+    @staticmethod
+    def _iter_references(obj: Any) -> Iterable[DependencyMixin]:
+        from airflow.models.baseoperator import AbstractOperator
+        from airflow.models.xcom_arg import PlainXComArg
+
+        if isinstance(obj, AbstractOperator):
+            yield obj
+        elif isinstance(obj, PlainXComArg):
+            yield obj.operator
+        elif isinstance(obj, Sequence):
+            for o in obj:
+                yield from DependencyMixin._iter_references(o)
```

Comment:
```suggestion
                yield from cls._iter_references(o)
```

if you change the decorator to `classmethod`

Reply:
I didn't know you can use `self` with classmethod. Thanks a bunch!